web/client/utils/FileFormatUtils.js

```diff
@@ -1,4 +1,10 @@
 const {head} = require('lodash');
+
+const getFormatByName = (outF) => {
+    const extension = outF.split('-')[1];
```

Comment:
I would split on "any non word char" to consider more special symbols.

outF.split(/[^\w]/)

Reply:
ok